README.md

```diff
@@ -57,22 +57,22 @@ By the end of the course, you will be able to ...
 |  6 | Wed, Sep 4  | Static Website <br/> Git & Github | [Intro to Flask] | Dani Roxberry | Meredith Murphy |
 |  7 | Thu, Sep 5  | -                             | [Flask Templating] | Dani Roxberry | Meredith Murphy |
 |  8 | Mon, Sep 9  | -                             | [Problem Solving Strategy] | Jess Dahmen | Alan Davis |
-|  9 | Tue, Sep 10 | -                             | [Code Quality, Reuse & Error Handling] | Jess Dahmen | Alan Davis |
-| 10 | Wed, Sep 11 | Pseudocode & Flowchart Diagrams <br/> Problem Solving Strategies <br/> Code Quality, Reuse & Error Handling | [Debugging & Testing] | Jess Dahmen | Alan Davis |
-| 11 | Thu, Sep 12 | Intro to Flask <br/> Flask Templating | [URLs, HTTP, REST & Error Handling] | Dani Roxberry | Meredith Murphy |
-| 12 | Mon, Sep 16 | -                             | [Testing RESTful Routes] | Dani Roxberry | Meredith Murphy |
-| 13 | Tue, Sep 17 | -                             | [OOP Part 1: Encapsulation] | Jess Dahmen | Alan Davis |
-| 14 | Wed, Sep 18 | URLs, HTTP, REST, & Reading Errors <br/> Testing RESTful Routes | [Document-based Databases] | Dani Roxberry | Meredith Murphy |
+|  9 | Tue, Sep 10 | -                             | [URLs, HTTP, REST & Error Handling] | Dani Roxberry | Meredith Murphy |
+| 10 | Wed, Sep 11 | -                             | [Code Quality, Reuse & Error Handling] | Jess Dahmen | Alan Davis |
+| 12 | Thu, Sep 12 | Intro to Flask <br/> Flask Templating | [Testing RESTful Routes] | Dani Roxberry | Meredith Murphy |
+| 11 | Mon, Sep 16 | Pseudocode & Flowchart Diagrams <br/> Problem Solving Strategies <br/> Code Quality, Reuse & Error Handling | [Debugging & Testing] | Jess Dahmen | Alan Davis |
```

Comment:
3 quiz topics in one day. Not saying you shouldn't do it, just something to consider when thinking about quiz length/number of questions